File: DAY6/EmployeeApp/EmployeeApp/Models/Employee.cs
namespace EmployeeApp.Models
{
    public class Employee
    {
        public int Id { get; set; }

        public string Password { get; set; }
        public required string FirstName { get; set; }
        public required string LastName { get; set; }
        public DateTime Birthdate{ get; set; }
        public double Salary { get; set; }
        public required string Email { get; set; }

    }
}

File: DAY6/EmployeeApp/employee-app-client/src/app/models/employee.ts
export interface EmployeeDto {
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  birthdate: string | Date;
  salary: number;
  age: number;
}

export interface CreateEmployeeRequest {
  firstName: string;
  lastName: string;
  email: string;
  birthdate: string | Date;
  salary: number;
}
